### PR TITLE
Display recipe neatly on a custom page

### PR DESCRIPTION
**Pull request by MichaelQuinonesCE**
Add six different recipes with links to each website (JS)
Add recipe layout (html)

Assigment given: Michael: Fake the data (JS side?), display recipe neatly on a custom page

### CODE REVIEW

**Review by sonambenakatti — APPROVED** [review]
Nice!! This is a great start before we get the database running.

**Review by GROD8962 — APPROVED** [review]
I concur; looks good as a prototype.

File: FetchRecipe.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function addRandomRecipe() {
+  const recipes =
```
Comment (anniefu) [inline FetchRecipe.js]: Great start!

I think the next step (for a new PR) is to generalize this formatting into a function that takes a recipe object as an input:

```
{
“id”: 0 
“link”: “some link”,
“title”: “some title”,
“like”: 5,
“category”: “some category”
}
```

And returns an HTML block that's formatted how you would like to see a recipe in the webpage. You could imagine the category's page might use it like this (pseudocode):
```
getRecipesByCategory() {
     var recipes = fetch('category-servlet');
     var recipeList = []
     for recipe in recipes {
         var formatted = formatRecipeToHTML(recipe);
         recipeList.add(formatted);
     }
    document.getElementById('category-page-recipe-list').innerText = recipeLIst;    
}
```

Similarly the frontpage might call the function too with different recipes. With a convenient formatting function, we can make sure that recipes look the same no matter where they are applied. Your function could also tag the HTML block with a CSS class so you can add styling through that too.

**Review by anniefu — CHANGES_REQUESTED** [review]
Hey Michael, I added some structuring to our repo similar to the examples throughout SPS so far, just so we all put files in the same place. Could you move these files to `src/main/webapp`?

Blocking on this change only.

**Review by anniefu — APPROVED** [review]

**Pull request merged**